irods/manager/user_manager.py

```diff
@@ -229,14 +283,27 @@ def addmember(self, group_name, user_name, user_zone=""):
             user_zone
         )
         request = iRODSMessage("RODS_API_REQ", msg=message_body,
-                               int_info=api_number['GENERAL_ADMIN_AN'])
+                               int_info=api_number[api_key])
         with self.sess.pool.get_connection() as conn:
             conn.send(request)
             response = conn.recv()
         logger.debug(response.int_info)
 
-    def removemember(self, group_name, user_name, user_zone=""):
-        message_body = GeneralAdminRequest(
+    # pending resolution of group admin rfg issue ( https://github.com/irods/irods/issues/6885 )
```

Comment:
Please somehow link this issue to the issue to which this change is related (and/or to this PR) in GitHub for future investigation

Reply:
Deleting the comment instead, as it was based on a lack of understanding.  `rfg` works fine, in the PRC as well, and always did.  I just wasn't aware a `groupadmin` had to belong to the group as a precondition.